src/libstd/sys/unix/stack_overflow.rs

```diff
@@ -219,6 +222,9 @@ mod imp {
         #[cfg(target_os = "openbsd")]
         pub const SIGSTKSZ: libc::size_t = 40960;
 
+        #[cfg(target_os = "bitrig")]
+        pub const SIGSTKSZ: libc::size_t = 40960;
+
```

Comment:
maybe collapse bitrig and openbsd in the same block ? but it is just cosmetic...